src/Command/MigrationsCacheClearCommand.php

```diff
@@ -0,0 +1,30 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://cakephp.org CakePHP(tm) Project
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Migrations\Command;
+
+/**
+ * This class is needed in order to provide a correct autocompletion feature
+ * when using the CakePHP migrations plugin. It has no effect on the
+ * migrations process.
+ *
+ */
+class MigrationsCacheClearCommand extends MigrationsCommand
```

Comment:
Do you need the `Migrations` class prefix when you are in a namespace already?

Reply:
since we are using `defaultName()` method , class name does not matter 
I can change it to `CacheClearCommand` or `CakeCacheClearCommand` to separate cake's command from symfony's command 

even if we able to send command name in `CompletionCommand` (core) to target command we can remove these Commands ( `MigrationsCacheClearCommand` , ......) 
but for now , no way to detect executed command by `CompletionCommand`

Reply:
Ok. We can trim the class names down later.